pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.github.sylvainlaurent.maven</groupId>
   <artifactId>yaml-json-validator-maven-plugin</artifactId>
-  <version>1.0.3-SNAPSHOT</version>
+  <version>1.0.2.1</version>
```

Comment:
please don't change the version, it's automatically done during the build of a release

Reply:
check, reverted it